content/en/docs/contributing/development.md

```diff
@@ -148,11 +166,31 @@ approvers and maintainers while doing code reviews:
 - Words unknown to cspell should be added to the cspell ignore list per page by
   PR authors. Only approvers and maintainers will add commonly used terms to the
   global list.
+- Approvers and maintainers have different work schedules and circumstances.
+  That's why all communication is assumed to be asynchronously and they should
+  not feel obligated to reply outside of their normal schedule.
 - When an approver or maintainer won't be available to contribute for an
   extended period of time (more than a few days or a week) or won't be available
   in that period of time, they should communicate this using the
   [#otel-comms](https://cloud-native.slack.com/archives/C02UN96HZH6) channel and
   updating the GitHub status.
+- Approver and maintainer adhere to the
+  [OTel Code of Conduct](https://github.com/open-telemetry/community/?tab=coc-ov-file#opentelemetry-community-code-of-conduct)
+  and are friendly and helpful towards contributors. In the case of a conflict,
+  misunderstanding or any other kind of situation that makes an
+  approver/maintainer feel uncomfortable they can step back from a conversation,
+  issue or PR and ask another approver/maintainer to step in.
+
+The following workflow can be applied by maintainers to merge PRs:
+
+- Make sure that a PR has all approvals and all CI checks pass
+- If the branch is out-of-date, update it via the GitHub UI.
```

Comment:
I'd suggest a rebase update. Merge updates can cause problems for long-running PRs.

Reply:
```suggestion
- If the branch is out-of-date, rebase update it via the GitHub UI.
```